Makefile.inc1

```diff
@@ -462,7 +462,7 @@ SUBDIR+=	${_DIR}
 # by calling 'makedb' in share/man.  This is only relevant for
 # install/distribute so they build the whatis file after every manpage is
 # installed.
-.if make(installworld) || make(install)
+.if make(installworld) || make(install) || make(distribute)
```

Comment:
Should this be `make(distributeworld)`? I'm not sure, but I don't fully understand the `distribute` pseduo-target.